doc/builtin_models.rst

```diff
@@ -640,3 +648,259 @@ and without any bounds on parameters at all:
 
 This script is in the file ``doc_builtinmodels_nistgauss2.py`` in the examples folder,
 and the figure above shows an improved initial estimate of the data.
+
+
+
+Example 4: Using a Spline Model
+===============================
```

Comment:
for consistency with the other titles in this document, you should use `-` here instead of `=`

Reply:
fixed